tinker/admin/cache/views.py

```diff
@@ -9,4 +10,14 @@ def before_request():
 
 @cache_blueprint.route('/')
 def home():
-    return render_template('cache-home.html', **locals())
\ No newline at end of file
+    return render_template('cache-home.html', **locals())
+
+@cache_blueprint.route('/submit', methods=['post'])
+def submit():
+    path = request.form['url']
+    return cache_clear(path)
+
+def cache_clear(img_path=None):
+    if not img_path:
+        img_path = '/academics/faculty/images/lundberg-kelsey.jpg'
```

Comment:
@nate-li can we have a better case here if they don't submit a path? Either a check in the JS or a more helpful error message returned from Python?